nbs/121b_models.GatedTabTransformer.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
<p>Hi @radi-cho,</p><p>This implementation looks good to me. </p><p>BTW, are you the author of the arxiv paper?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/timeseriesAI/tsai/pull/365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Thank you!
Yes, I am the author.